timeutils.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from datetime import timedelta, timezone, datetime
+logger = logging.getLogger(__name__)
+
+# Nowy York xD
+MEETUP_TZ = timezone(-timedelta(hours=4))
```

Comment:
???
That's confusing...

Reply:
Reference const, Meetup is NYC based company. It looks like timezone shouldn't matter after all.

Reply:
@not7cd I mean -timedelta

Reply:
https://docs.python.org/3/library/datetime.html#datetime.timezone